.travis.yml

```diff
@@ -1,17 +1,27 @@
 sudo: false
 language: python
-python:
-  - 2.6
-  - 2.7
-  - 3.3
-  - 3.4
-  - 3.5
+matrix:
+  include:
+    - python: 2.6
+      env: TOXENV=py26
+    - python: 2.7
+      env: TOXENV=py27
+    - python: 3.3
+      env: TOXENV=py33
+    - python: 3.4
+      env: TOXENV=py34
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 3.6
+      env: TOXENV=py36
+    - python: pypy
+      env: TOXENV=pypy
+    - python: pypy3
+      env: TOXENV=pypy3
+
 install:
-    - if [[ $TRAVIS_PYTHON_VERSION == '2.6' ]]; then make build26; fi
-    - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then make build27; fi
-    - if [[ $TRAVIS_PYTHON_VERSION == '3.3' ]]; then make build33; fi
-    - if [[ $TRAVIS_PYTHON_VERSION == '3.4' ]]; then make build34; fi
-    - if [[ $TRAVIS_PYTHON_VERSION == '3.5' ]]; then make build35; fi
-    - pip install codecov
-script: make test
-after_success: codecov
+    - pip install tox
+script: tox
+after_success:
+  - pip install codecov
```

Comment:
Does Travis CI cache the `install` step for future builds?  If so, we should install codecov in the install step.

Reply:
You can cache like this:
https://docs.travis-ci.com/user/caching/#pip-cache